### PR TITLE
[Merged by Bors] - chore: update/remove heart beat bumps

### PR DESCRIPTION
**Pull request by mattrobball**
We clean up heart beat bumps after #6474.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/InnerProductSpace/Spectrum.lean [hunk 1]
```diff
@@ -139,7 +139,6 @@ theorem orthogonalComplement_iSup_eigenspaces_eq_bot' :
 #align linear_map.is_symmetric.orthogonal_supr_eigenspaces_eq_bot' LinearMap.IsSymmetric.orthogonalComplement_iSup_eigenspaces_eq_bot'
 
 -- porting note: a modest increast in the `synthInstance.maxHeartbeats`, but we should still fix it.
```
Comment (ericrbg) [inline Mathlib/Analysis/InnerProductSpace/Spectrum.lean:141]:
```suggestion
```

File: Mathlib/Analysis/Convolution.lean [hunk 2]
```diff
@@ -1198,7 +1198,7 @@ variable [IsROrC 𝕜] [NormedSpace 𝕜 E] [NormedSpace 𝕜 E'] [NormedSpace 
   (L : E →L[𝕜] E' →L[𝕜] F)
 
 -- porting note: the lemma is slow, added `set_option maxHeartbeats 300000 in`
```
Comment (ericrbg) [inline Mathlib/Analysis/Convolution.lean:1200]:
```suggestion
-- porting note: the lemma is slow, added `set_option maxHeartbeats 250000 in`
```

**Review by ericrbg — COMMENTED** [review]
not sure if all of these comments should be removed, but just in case they were missed:

**Comment by mattrobball** [comment]
I should have caught those all with the others I deleted.

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**